src/Domain/FileProcessors/FixerFileProcessor.php

```diff
@@ -59,18 +60,23 @@ public function processFile(SplFileInfo $splFileInfo): void
 
         $oldContent = $splFileInfo->getContents();
 
-        $tokens = Tokens::fromCode($oldContent);
-        /** @var FixerDecorator $fixer */
-        foreach ($this->fixers as $fixer) {
-            $fixer->fix($splFileInfo, $tokens);
-            if (! $tokens->isChanged()) {
-                continue;
-            }
-
-            $fixer->addDiff($filePath, $this->differ->diff($oldContent, $tokens->generateCode()));
-            // Tokens has changed, so we need to clear cache
-            Tokens::clearCache();
+        try {
             $tokens = Tokens::fromCode($oldContent);
+            /** @var FixerDecorator $fixer */
+            foreach ($this->fixers as $fixer) {
+                $fixer->fix($splFileInfo, $tokens);
+                if (! $tokens->isChanged()) {
+                    continue;
+                }
+
+                $fixer->addDiff($filePath,
+                    $this->differ->diff($oldContent, $tokens->generateCode()));
+                // Tokens has changed, so we need to clear cache
+                Tokens::clearCache();
+                $tokens = Tokens::fromCode($oldContent);
+            }
+        } catch (\Throwable $e) {
+            // @ignoreException
```

Comment:
Why are we ignoring exceptions? Can you give context? 👍

Reply:
Assume we have below code in one of our files: (as explained in issue #322)

```
<?php

class MyClass extends Illuminate\Database\Eloquent\Model; {
```
And this code cause throwing fatal error => syntax error, unexpected ';', expecting '{' .
For avoiding this error and terminating "insight", we should ignor this exceptions and add them to "insight" error report.

Reply:
I see. Shouldn't we display something instead? Like error while analysing blabla?

Reply:
We can do that. But "insight will be terminated". This way we gave the user this error in final report ("Unparsable php code: syntax error or wrong phpdocs.")